deploy/vagrant/provision/roles/spark/files/remove_s3_exclusion.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+python <<EOF
+import re
+
+PATTERN = r'''<exclusion>
+\s*<groupId>org\.tachyonproject</groupId>
+\s*<artifactId>tachyon-underfs-s3</artifactId>
+\s*</exclusion>'''
+POM = '/spark/core/pom.xml'
+
+new_pom = re.compile(PATTERN).sub('', open(POM).read())
+open(POM, 'w').write(new_pom)
+EOF
```

Comment:
Feasible, but then we need to manually send the `.py` to remote node. When the script is `.sh`, ansible will send the script to remote node and execute. Since the python script is simple enough, I'd like to keep this form.